### PR TITLE
Patch/strider plater

### PR DESCRIPTION
**Pull request by YaphetKG**
- Ingress config to avoid upstream disconnected before sending data (strider and aragorn)
- Neo4j stack size config
- Strider memory bump
- Neo4j + plater query timeout (30mins)
- Neo4j service type Nodeport  by default

### CODE REVIEW

File: helm/plater/values.yaml [hunk 1]
```diff
@@ -7,7 +7,7 @@ replicaCount: 1
 image:
   plater:
     repository: renciorg/plater-clustered
-    tag: "1.1.2-1"
+    tag: "1.1.2-2rc"
```
Comment (PhillipsOwen) [inline helm/plater/values.yaml:10]: are you going to leave this as a release candidate?

Comment (YaphetKG) [reply helm/plater/values.yaml:10]: opps sorry reverting back

Comment (YaphetKG) [reply helm/plater/values.yaml]: should be fixed now

**Review by PhillipsOwen — APPROVED** [review]
looks good!

**Pull request merged**